cartridge/lua-api/boxinfo.lua

```diff
@@ -51,8 +51,20 @@ local function get_info(uri)
         local membership_myself = require('membership').myself()
         local membership_options = require('membership.options')
 
+        local vshard, vshard_version, ddl_version
+        if package.loaded['vshard-ee'] ~= nil then
```

Comment:
Why not require here?

Reply:
package.loaded was used in the previous version of this code. I can change it to require if necessary.

Reply:
I think it would be safer to use `require`, especially in case of ddl version extraction

Reply:
I found out the reason why package.loaded was used. In some tests vshard might be disabled. Rare case, but I'd prefer to leave vshard with package.loaded. 
But I'll change package.loaded['ddl'] to package.loaded['ddl.version']

Reply:
`package.loaded['module.version']` and `require('module.version')` are antipatterns. Public API is `require('module')._VERSION`, and `require('module.version')` is just an internal implementation of this API (even though I don't remember any modules which use some other implementation).

Isn't `require('vshard')` should always be a success for regular cartridge since it is a rock dependency?

Reply:
Fixed